batch/indexer_Token_Holders.py

```diff
@@ -167,11 +167,12 @@ def collect(self):
             self.__process_all(local_session, self.block_from, self.block_to)
             self.__update_status(local_session, TokenHolderBatchStatus.DONE)
             local_session.commit()
-        except Exception:
+        except Exception as e:
             LOG.exception("An exception occurred during event synchronization")
             local_session.rollback()
             self.__update_status(local_session, TokenHolderBatchStatus.FAILED)
             local_session.commit()
+            raise e
         finally:
             local_session.close()
             LOG.info(f"<{process_name}> Collect job has been completed")
```

Comment:
I wondered if I should move the session.commit() into the finally clause.
If I did so, processor would not be able to handle exceptions that occurred within finally clause(eg. DB error),
so I implemented as above.